RadixWallet/Clients/OnLedgerEntitiesClient/Helpers/OnLedgerEntitiesClient+CreateEntity.swift

```diff
@@ -564,9 +494,15 @@ extension OnLedgerEntitiesClient {
 
 	public struct OwnedResourcePoolDetails: Hashable, Sendable {
 		public let address: ResourcePoolAddress
+		public let dAppName: String?
 		public let poolUnitResource: ResourceWithVaultAmount
-		public let xrdResource: ResourceWithVaultAmount?
-		public let nonXrdResources: [ResourceWithVaultAmount]
+		public let xrdResource: ResourceWithRedemptionValue?
+		public let nonXrdResources: [ResourceWithRedemptionValue]
+
+		public struct ResourceWithRedemptionValue: Hashable, Sendable {
+			public let resource: OnLedgerEntity.Resource
+			public let redemptionValue: String
```

Comment:
I still don't like that this is a string, isn't it breaking error if the total supply is missing for example? Should we really recover from that? And if we should, then I think this should be either an optional `RETDecimal`, a `Result<RETDecimal>` or an `Either`, and the same for other places. Storing values as strings is only acceptable in ViewState in my view.

Another thing, should this really be called `**Owned**ResourcePoolDetails`?